src/OrchardCore.Modules/OrchardCore.Deployment/Controllers/ImportController.cs

```diff
@@ -146,7 +165,19 @@ public async Task<IActionResult> Json(ImportJsonViewModel model)
 
                     await _deploymentManager.ImportDeploymentPackageAsync(new PhysicalFileProvider(tempArchiveFolder));
 
-                    await _notifier.SuccessAsync(H["Recipe imported."]);
+                    await _notifier.SuccessAsync(H["Recipe imported successfully!"]);
+                }
+                catch (RecipeExecutionException e)
+                {
+                    _logger.LogError(e, "Unable to import a recipe from JSON input.");
```

Comment:
We could store the log message into a variable while its used in two places

Reply:
Different message is used not the same

Reply:
If not same ignore this, seems I saw similar messages down below :)